try:
    personas = ['Juan', '√Åndres', 'Camilo']
    print(personas[5])
except IndexError:
    print("Error: Este dato no existe")